pallets/capacity/src/benchmarking.rs

```diff
@@ -125,7 +139,47 @@ benchmarks! {
 		assert_last_event::<T>(Event::<T>::EpochLengthUpdated {blocks: epoch_length}.into());
 	}
 
+	change_staking_target {
```

Comment:
Since the weights are not generated for this it is not applied on top of `change_staking_target` function. You should be able to generate the weight locally for that function to be able to add it on top of your function and then later it will be rewritten with correct weights from CI if generated

Reply:
thanks for the reminder... it's been a minute!

Reply:
Done